src/emulator/functionsEmulator.ts

```diff
@@ -641,6 +651,10 @@ export class FunctionsEmulator implements EmulatorInstance {
     const childProcess = spawn(opts.nodeBinary, args, {
       env: { node: opts.nodeBinary, ...opts.env, ...process.env },
       cwd: frb.cwd,
+      // By running as detached the child proccesses don't pick up
+      // kill signals to the host. This allows functions to finish
+      // running in workqueue.flush()
+      detached: true,
```

Comment:
This is the part that scares me ... is this tradeoff worth it?  The workqueue should still kill all the child processes during `stop()` but there's always a chance it explodes and leaves zombies.

Reply:
I haven't really reviewed the rest of the PR, but wild idea:

Instead of spinning these up detached, can we inject SIGINT handlers into the child processes instead, so they gracefully shutdown (and maybe force shutdown when pressed twice)?

We may have to do [`node -r`](https://nodejs.org/api/cli.html#cli_r_require_module) or put more logic into the testing module or require hooks, but I think that scares me less than having detached processes run loose.

Reply:
@yuchenshi great idea!  I installed a signal handler in the runtime and it works perfectly.